src/OrbitGl/GpuSubmissionTrack.cpp

```diff
@@ -124,7 +125,7 @@ Color GpuSubmissionTrack::GetTimerColor(const TimerInfo& timer_info, bool is_sel
 
 float GpuSubmissionTrack::GetYFromTimer(const TimerInfo& timer_info) const {
   auto adjusted_depth = static_cast<float>(timer_info.depth());
-  if (collapse_toggle_->IsCollapsed()) {
+  if (IsCollapsed() || GetParent()->IsCollapsed()) {
```

Comment:
I suggest adding an helper method, e.g., `ShouldShowCollapsed`, and using it in these three occurrences.